.gitignore

```
@@ -4,4 +4,8 @@
 *AWSCLIV2.pkg
 *.egg-info/
 venv
-.venv
\ No newline at end of file
+.venv
+.DS_Store
+
+# Model Weights
+brainscore_vision/models/**/*.pt
```

```suggestion
*.pt
```
I think we can remove `.pt` files altogether